salt/states/pkg.py

```diff
@@ -2678,6 +2678,18 @@ def mod_aggregate(low, chunks, running):
                     elif 'name' in chunk:
                         version = chunk.pop('version', None)
                         if version is not None:
+                            if version == 'latest':
+                                try:
+                                    version = __salt__['pkg.latest_version'](chunk['name'],
+                                                                             fromrepo=chunk.get('fromrepo'),
+                                                                             refresh=chunk.get('refresh'))
```

Comment:
Yeah, I'm not so sure that this is the correct fix though... when we aggregate the packages, it still processes them through ``pkg.installed``. So the proper fix should be to support ``latest`` in ``_find_install_targets``.